### PR TITLE
Added JSDocs to cheap-ruler

### PR DESCRIPTION
**Pull request by DenisCarriere**
@tmcw @mourner please review to make sure the JSDocs is accurate.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,11 +2,29 @@
 
 module.exports = cheapRuler;
 
+/**
+ * A collection of very fast approximations to common geodesic measurements. Useful for performance-sensitive code that measures things on a city scale.
+ *
+ * @name cheapRuler
+ * @param {number} lat latitude
+ * @param {string} [units='kilometers']
+ * @returns {Object} CheapRuler
```
Comment (tmcw) [inline index.js]: Instead of `Object` here, could we specify `CheapRuler`?

File: index.js [hunk 2]
```diff
@@ -2,11 +2,29 @@
 
 module.exports = cheapRuler;
 
+/**
+ * A collection of very fast approximations to common geodesic measurements. Useful for performance-sensitive code that measures things on a city scale.
+ *
+ * @name cheapRuler
```
Comment (tmcw) [inline index.js]: Are the `@name` tags necessary? Ideally both JSDoc and documentation.js should both be able to infer naming, and importantly name [turns off inference](http://usejsdoc.org/tags-name.html) in both engines, so mistakes like misnamed parameters won't be detected.

Comment (DenisCarriere) [inline index.js]: That makes sense, you know more about JSDocs and `documentation.js` engines.

File: index.js (hunk 1 above)
Comment (DenisCarriere) [inline index.js]: That looks better.

**Comment by mourner** [comment]
Thanks for taking time to contribute! I'm generally not a fan of JSDoc, but it makes sense in this repo.

**Comment by DenisCarriere** [comment]
@mourner I can tell that you're not a fan :) The hardest part is the initial documentation and you've done a great job defining it which I thank you! I wrote the JSDocs for the TypeScript definition I just figured it would be best to also match those same JSDocs in your source code.

You've got a lot more libraries I want to tackle, you're libraries are awesome! Very clean work.

**Pull request merged**